src/components/Card/Card.tsx

```diff
@@ -27,20 +28,20 @@ const Card = ({
               text="Open"
               variant="secondary"
               onClick={() => {
-                router.push(`/ofd/id/${data.id}`);
+                router.push(`/ofd/id/${id}`);
               }}
             ></tds-button>
           </div>
         </div>
-        <h3 className={styles.card__top__name}>{data.name}</h3>
-        <p className={styles.card__top__description}>{data.description}</p>
+        <h3 className={styles.card__top__name}>{name.split("/").pop()}</h3>
```

Comment:
This will not work correctly.

If I name a graph "foo/bar", save it, and then check the cards on the landing page, it is named "bar". We could add rules to what the user can name their graph..? I guess there are a couple of solutions, and if you dont wish to solve it in this MR, then create a Jira for it so that we can track the issue.